src/main/include/subsystems/Shooter.h

```diff
@@ -48,6 +52,8 @@ namespace t34
 
         void SetMaxSpeedPercent(const double percent);
 
+        double GetMaxSpeedPercent();
```

Comment:
Suggest tagging with `const`:

```suggestion
        double GetMaxSpeedPercent() const;
```